concent_api/core/models.py

```diff
@@ -518,3 +519,39 @@ def clean(self):
             raise ValidationError({
                 'pending_response': 'PaymentInfo should be related with Pending Response'
             })
+
+
+class GlobalTransactionState(Model):
+    """
+    Represents state of whole transaction service by storing last `nonce` used.
+
+    There should always be at most one object of this type with id = 0. If it does not exist, it should be created and
+    nonce initialized with a value obtained from SCI (Client.get_transaction_count()).
+    """
+
+    nonce = DecimalField(max_digits=32, decimal_places=0, default=0, unique=True)
+
+    def save(self, *args, **kwargs):  # pylint: disable=arguments-differ
+        self.pk = 0
+        super().save(*args, **kwargs)
+
+
+class PendingEthereumTransaction(Model):
+    """
+    Represents pending Ethereum transaction state.
+    """
+
+    nonce = DecimalField(max_digits=32, decimal_places=0, unique=True)
+
+    gasprice = CharField(max_length=32)
+    startgas = CharField(max_length=32)
+    value = CharField(max_length=32)
+
+    to = BinaryField(max_length=20)
```

Comment:
Field `to` in `transaction.json` is stored as a string.

Reply:
That's most probably because JSON doesn't support binary data, so it has to be converted to string. I think our point is to be able to store and recreate `Transaction` to be exactly the same, not storing the data in the same format as `JsonTransactionsStorage`, but if I am wrong then please correct me. @cameel ?

Btw. 'I think our point is to be able to store and recreate `Transaction`' is a good scenario for test, I will add it.

Reply:
You're right. We want to store the original transaction, not a transaction mangled by being converted to JSON.